#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#       name    type     level     description     default      min      max
gen.add("synchronization", bool_t, 0, "Enable Synchronization", False)
gen.add("approximate_sync", bool_t, 0, "Approximate synchronization", False)
gen.add("queue_size", int_t, 0, "Queue size", 1, 1, 1000)
gen.add("crop_size", int_t, 0, "Size of cropping", 10, 1, 100)
gen.add("search_size", int_t, 0, "Size of search", 16, 1, 100)
gen.add("timeout", double_t, 0, "Timeout to wait for point cloud", 3.0, 0.0, 30.0)
exit(gen.generate("jsk_pcl_ros", "jsk_pcl_ros", "PointcloudScreenpoint"))
